Source/RxSwift/and.swift

```diff
@@ -0,0 +1,22 @@
+extension Observable where E == Bool {
+
+   func and(_ right: Observable<Bool>) -> Observable<Bool> {
```

Comment:
I think these should be `Observable<E>`, right?

Reply:
Correct, I will refactor it 👍